libs/application/templates/new-primary-school/src/fields/Review/review-groups/School.tsx

```diff
@@ -62,43 +68,31 @@ export const School = ({
         ) : (
           <>
             <GridRow rowGap={2}>
-              <GridColumn span={['12/12', '12/12', '12/12', '5/12']}>
+              <GridColumn span={['12/12', '12/12', '12/12', '12/12']}>
```

Comment:
This should be like this was
```suggestion
              <GridColumn span={['12/12', '12/12', '12/12', '5/12']}>
```

Reply:
Fixed